CorsixTH/Lua/utility.lua

```diff
@@ -279,7 +279,7 @@ function compare_tables(t1, t2)
     if t2[k] ~= v then return false end
   end
   local count2 = 0
-  for k, v in pairs(t2) do
+  for _, _ in pairs(t2) do
```

Comment:
Would it be better to remove count1, too and use
```
if #t1 ~= #t2 then return false end
```
instead of
```
if count1 ~= count2 then return false end
```